template.yaml

```diff
@@ -373,6 +399,29 @@ Resources:
           KINESIS_STREAM_ARN: !Ref KinesisStreamArn
           ADMIN_ROLE_NAME: !Ref AdminRoleName
           MASTER_MGM_AWS_ID: !Ref MasterMgmAwsId
+
+  GcloudProxyLambdaFunction:
```

Comment:
We will modify proxy function and will add a code to fetch the G-Cloud DB password. Do we  really need separate proxy lambda function?

Reply:
creating separate lambda function will definitely will provide us the advantage of using same env variables and keep our logic same without making much changes by writing if else conditions. Also we will able to make decision to fetch password and invoke state machines based on env variables.